.github/workflows/release-deletion.yml

```diff
@@ -0,0 +1,32 @@
+#
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+# This workflow disables release/* branch protection and deletes the branch after merge to develop
+name: "release deletion"
+on:
+  pull_request:
+    types: [closed]
+    branches: [main]
+
+jobs:
+  delete-release-branch-on-merge:
```

Comment:
can the release branch live past the merge?

Reply:
We are merging it using the `gh api`, so we need to delete it post merge to `main`

Reply:
this is more of a concern if there is a need of maintaining a release post merging to main

Reply:
Post final merge the branch should be deleted like any other branch. If we need to make fixes we can always create a new release branch from main post merge